streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -595,6 +595,7 @@ public boolean create(Application appParam) {
         appParam.setLaunch(LaunchState.NEED_LAUNCH.get());
         appParam.setOptionState(OptionState.NONE.getValue());
         appParam.setCreateTime(new Date());
+        appParam.setDefaultModeIngress(settingService.getIngressModeDefault());
```

Comment:
If the user does not set `IngressModeDefault`, then set a null here, will an exception occur?

Reply:
no problem